common/iprinter.ml

```diff
@@ -239,6 +239,7 @@ let rec string_of_formula_exp = function
   | P.ListTail (e, l)		-> "tail(" ^ (string_of_formula_exp e) ^ ")"
   | P.ListLength (e, l)		-> "len(" ^ (string_of_formula_exp e) ^ ")"
   | P.ListReverse (e, l)	-> "rev(" ^ (string_of_formula_exp e) ^ ")"
+  | P.ExpPack (elist, l)		-> "pack[" ^ (string_of_formula_exp_list elist) ^ "]"
```

Comment:
Just curious, where is this name from?

Reply:
It came from the notion of a "pack" as a list of parameters elsewhere (e.g. "unpacking" in Python, "template parameter pack" in C++). Do you have any more intuitive names in mind?

Reply:
I see, good to know. What came to mind initially when reading it was the pack/unpack operations for existential types. I don't have a better suggestion.